src/common/services/registerAccount.service.js

```diff
@@ -0,0 +1,86 @@
+import angular from 'angular';
+import orderService from 'common/services/api/order.service';
+import sessionModalService from 'common/services/session/sessionModal.service';
+import sessionService from 'common/services/session/session.service';
+
+import {Roles} from 'common/services/session/session.service';
+
+let serviceName = 'registerAccountService';
+
+/*@ngInject*/
+function RegisterAccountService( $q, orderService, sessionService, sessionModalService ) {
+
+// angular.element(document.querySelector('body')).injector().get('registerAccountService')().then(function() {console.log('success');}, function() {console.log('failure')});
```

Comment:
You want to add a comment to indicate this code is for manually opening the modal? (I assume that's what it is anyways.) Just so it doesn't look like commented out code that someone might remove and so people know how to use it in the future.